src/main/java/org/mskcc/cmo/publisher/pipeline/MetaDbFilePublisherReader.java

```diff
@@ -28,7 +29,7 @@
 
     private List<Map<String, String>> messagesToPublish;
 
-    private Logger LOG = Logger.getLogger(MetaDbFilePublisherReader.class);
+    private static final Log LOG = LogFactory.getLog(MetaDbFilePublisherReader.class);
```

Comment:
Should this log be static?

Reply:
Also do we still need log4j properties file? Can i go ahead and remove it?

Reply:
@divyamadala30 oh yeah good catch, thanks for reminding me